src/home/components/home-banner/HomeBanner.tsx

```diff
@@ -53,7 +52,7 @@ const HomeBanner = () => {
       <HomePromoVideoModalSlot
         isOpen={isOpen}
         close={close}
-        videoID={getConfig().HOMEPAGE_PROMO_VIDEO_YOUTUBE_ID || ''}
+        videoId={getConfig().HOMEPAGE_PROMO_VIDEO_YOUTUBE_ID || ''}
```

Comment:
[inform]: I changed `videoID` to `videoId` in the project code because I think the short name of the identifier is not an abbreviation that should be explicitly specified in the variable name.